portal/resources/views/admin/voucher/clients.blade.php

```diff
@@ -0,0 +1,24 @@
+@extends('layouts.admin')
+
+@section('content')
+<div class="container">
+    <table class="table">
+        <thead>
+            <tr>
+                <th>Name</th>
+                <th>MAC</th>
+                <th>End date</th>
+            </tr>
+        </thead>
+        <tbody>
+            @foreach ($clients as $client)
+            <tr>
+                <td>{{$client->name}}</td>
+                <td>{{$client->mac}}</td>
+                <td>{{$client->endDate->format('Y-m-d H:i:s')}}</td>
```

Comment:
Optional: Can be replaced with Carbon method `->toDateTimeString()`

```
<td>{{$client->endDate->toDateTimeString()}}</td>
```

Reply:
Not a trivial option, since its not a Carbon date.